src/app/utils/iac.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Contains utility functions for IaC migration to support both old (pre-IaC) and new (IaC) environments.
+ */
+// TODO: (IaC Migration) Delete and remove all usage of thie file after IaC migration is fully completed
+import fs from 'fs'
+
+import config from '../config/config'
+
+const isIacMigrated = config.iacMigration.isMigrated
+
+/**
+ * Retrieves the content of a file.
+ * If the environment is post-IaC, the file content has been converted to base64 and stored in the SSM param store directly.
+ * If the environment is pre-IaC, the file content is stored in the file system and needs to be read via the file path.
+ *
+ * Relies on convict validation to ensure that the arguments are not null based on the NODE_ENV.
+ * @param preIacFilePath - The path to the file in the pre-IaC environment.
+ * @param postIacBase64EncodedString - The base64 encoded string of the file in the IaC environment.
+ * @returns The content of the file.
+ */
+export const retrieveFileContent = ({
+  preIacFilePath,
+  postIacBase64EncodedString,
+}: {
+  preIacFilePath: string
+  postIacBase64EncodedString: string
+}): string => {
+  if (isIacMigrated) {
+    return Buffer.from(postIacBase64EncodedString, 'base64').toString('utf-8')
```

Comment:
Is there a reason to encode the fileContent when we add it to SSM? Previously this wasn't encoded, and thus no need to have a decode step.

Reply:
This is since SSM param store types only consist `String`, `StringList`, `SecureString`, which hints that SSM param store intends for the value to be of type string. 

Many of the .pem, .key files etc are in binary format, which SSM param store does not support. Hence, I opted to convert this into string to match what the param store desires and used base64 as a consistent encoding function regardless of binary file type.

Reply:
Added the rationale in the docs for future dev reference

Reply:
As discussed, copying for posterity, most keys are generated as an encoded Base64 for transport thus there's likely no need for us to manually encode/decode this.

Noted on checking back on this when we're operating for `production`.